myNUnit/myNUnit/Assert.cs

```diff
@@ -0,0 +1,12 @@
+namespace myNUnit;
```

Comment:
```suggestion
namespace MyNUnit;
```
Тут и ниже. Стайлгайд же.